abci/example/kvstore/persistent_kvstore.go

```diff
@@ -447,3 +447,16 @@ func (app *PersistentKVStoreApplication) updateQuorumHash(
 
 	return types.ResponseDeliverTx{Code: code.CodeTypeOK}
 }
+
+func copyValidatorSetUpdate(vsu types.ValidatorSetUpdate) types.ValidatorSetUpdate {
```

Comment:
Maybe it's better to make it a method of ValidatorSetUpdate object?  Looks like we use the method approach in other parts of code.